pkg/arvo/lib/group-json.hoon

```diff
@@ -0,0 +1,30 @@
+/-  *group-store
+=,  enjs:format
+|%
+++  groups-to-json
+  |=  grp=group-initial
+  =,  enjs:format
+  ^-  ^json
+  %+  frond  %group-initial
+  %-  pairs
+  %+  turn  ~(tap by grp)
+  |=  [=path =group]
+  ^-  [@t ^json]
+  :-  (spat path)
+  (sa group ship:enjs:format)
+::
+++  sa                                                :::  set as array
```

Comment:
I assume `+sa`, `+as`, and `+pa` will be removed from here once that lands in zuse.

Reply:
Yup.